platform/web/js/engine/engine.externs.js

```diff
@@ -1,4 +1,4 @@
-var Godot;
+var Blazium;
```

Comment:
If this is not user facing, it should probably not be changed so we can avoid future merge conflicts.

Reply:
![image](https://github.com/user-attachments/assets/14ee2078-3295-4e1c-87e9-aef8fd5fc017)

Reply:
If it is a required change, then that is fine. Like I said, I am not a web dev.